hack/test-e2e-from-ci-bundle.sh

```
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+current_dir=$(dirname "${BASH_SOURCE[0]}" )
+GOFLAGS=-mod=mod NO_COLOR=1 go test -v -timeout=90m -ginkgo.v -ginkgo.trace \
+   -ginkgo.poll-progress-after=300s \ -ginkgo.poll-progress-interval=30s \
+   ${current_dir}/../test/e2e/...
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)